packages/foreman/foreman/foreman.spec

```diff
@@ -885,22 +883,23 @@ cp -r %%{%{name}_dir} ./%{_datadir} || echo 0 \\
 mkdir -p ./%{_localstatedir}/lib/%{name} \\
 cp -r %{_localstatedir}/lib/%{name}/db ./%{_localstatedir}/lib/%{name} || echo 0 \\
 unlink ./%{_datadir}/%{name}/db \\
-ln -sv \`pwd\`/%{_localstatedir}/lib/%{name}/db ./%{_datadir}/%{name}/db \\
 pushd ./%%{%{name}_dir} \\
+mkdir db/ \\
+cp -rf %{_datadir}/%{name}/db/* db/ \\
+mv db/schema.rb.nulldb db/schema.rb
 \\
 ln -s %{?scl_prefix_nodejs:%{_scl_root}}%{nodejs_sitelib} node_modules \\
 export GEM_PATH=%%{buildroot}%%{gem_dir}:\${GEM_PATH:+\${GEM_PATH}}\${GEM_PATH:-\`%{?scl:scl enable %%{scl} -- }ruby -e "print Gem.path.join(':')"\`} \\
 unlink tmp \\
 \\
 rm \`pwd\`/config/initializers/encryption_key.rb \\
+rm \`pwd\`/config/database.yml \\
```

Comment:
what would have created this file?

Reply:
This line copies the example to `database.yml` so there is a default: https://github.com/theforeman/foreman-packaging/pull/5058/files#diff-3399e5c03dc282964481f8f3ced9c141R796

When we drop SQlite we need to re-think this but for now we are just switching the nulldb usage.

Reply:
that link goes nowhere, stupid GH?